src/camera.js

```diff
@@ -609,9 +706,54 @@ vgl.camera = function () {
     return m_projectionMatrix;
   };
 
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * Convert a zoom level and window size to a camera height.
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  this.zoomToHeight = function (zoom, width, height) {
+    return vgl.zoomToHeight(zoom, width, height, m_viewAngle);
+  };
+
   this.computeDirectionOfProjection();
 
   return this;
 };
 
 inherit(vgl.camera, vgl.groupNode);
+
+////////////////////////////////////////////////////////////////////////////
+/**
+ * Convert a zoom level and window size to a camera height.
+ *
+ * @param zoom: Zoom level, as used in OSM maps.
+ * @param width: width of the window.
+ * @param height: height of the window.
+ * @returns: perspective camera height.
+ */
+////////////////////////////////////////////////////////////////////////////
+vgl.zoomToHeight = function (zoom, width, height, viewAngle) {
```

Comment:
I agree.  There needs to be someway for the camera to know the pixel size, and there should be a *single* class where we have this function.  Prior to this change, this calculation was done in several disparate locations.